src/ReportManager/SchemaFormContext/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { createContext } from 'react';
```

Comment:
initially was like that, but I was mixing responsibilities, let me organize it as it was

Reply:
If we are able to hide the context under the form component (which we should), it will be way clearer to simply call `SchemaForm` and keep everything under it as implementation details. Btw, `SchemaForm` is the definitive name we decided to differentiate old from new versions? Just asking, kind of like it.

Reply:
That is right, that was my intention to hide implementation behind `<SchemaForm />` the file you were sawing was a WIP for something I tried, but at the end I returned to the initial approach, sorry about that I forgot to mentioned it

Reply:
And yes, SchemaForm is the name I'm proposing haha